<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adRowDialog</class>
 <widget class="QDialog" name="adRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>222</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Table:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tableName">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="insertButton">
            <property name="text">
             <string>Insert Runs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="removeButton">
            <property name="text">
             <string>Remove Runs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="atTopButton">
            <property name="text">
             <string>At Top</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="atBottomButton">
            <property name="text">
             <string>At Bottom</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="afterButton">
              <property name="text">
               <string>After Run</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="aftetsb"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>No. of Runs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="noRunsb"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="OKButton">
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>OKButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>insertButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>atTopButton</tabstop>
  <tabstop>atBottomButton</tabstop>
  <tabstop>afterButton</tabstop>
  <tabstop>aftetsb</tabstop>
  <tabstop>noRunsb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
